### PR TITLE
Workaround for postgres issue on dev cluster

### PR DESCRIPTION
**Pull request by rhopp**
https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2570

### CODE REVIEW

**Comment by ibuziuk** [comment]
@rhopp PR check is failing but now on tests: https://ci.centos.org/job/devtools-rh-che-rh-che-prcheck-dev.rdu2c.fabric8.io/1215/console

> Caused by: java.lang.Exception: Output: ; Error: must specify at least one container source.

ideas ?

**Comment by amisevsk** [comment]
Looks like actual CI error is 

```
[pool-1-thread-1] INFO com.redhat.che.selenium.core.workspace.CheStarterWrapper - Workspace was started. Waiting until workspace is running.
[pool-1-thread-1] ERROR com.redhat.che.selenium.core.client.RhCheTestWorkspaceServiceClient - Failed to start workspace "web-nodejs-sample-lwjld".
[pool-1-thread-1] ERROR com.redhat.che.selenium.core.workspace.RhCheTestWorkspaceImpl - Could not start workspace with name: web-nodejs-sample-lwjld
[pool-1-thread-1] ERROR org.eclipse.che.selenium.core.inject.SeleniumTestHandler - Test WorkspaceTest.testCorrectWayOfStarting failed. Error: Expected condition failed: waiting for 300 SECONDS with 1000 MILLISECONDS interval
```
so workspace isn't reaching running state?

**Comment by rhopp** [comment]
[che-master logs](https://pastebin.com/Fad42u83)
Most noticeably: 
```
2019-03-04 16:34:50,760[aceSharedPool-0]  [WARN ] [.i.k.KubernetesInternalRuntime 249]  - Failed to start Kubernetes runtime of workspace workspacevjpm0dl55rwaicpz. Cause: Failure executing: GET at: https://osoproxy.prod-preview.openshift.io/api/v1/namespaces/rh-che-automation-preview-che/pods/mkdir-workspacevjpm0dl55rwaicpz/log?pretty=false. Message: container "mkdir-workspacevjpm0dl55rwaicpz" in pod "mkdir-workspacevjpm0dl55rwaicpz" is not available. Received status: Status(apiVersion=v1, code=400, details=null, kind=Status, message=container "mkdir-workspacevjpm0dl55rwaicpz" in pod "mkdir-workspacevjpm0dl55rwaicpz" is not available, metadata=ListMeta(_continue=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=BadRequest, status=Failure, additionalProperties={}).
```

**Comment by rhopp** [comment]
[test]

**Comment by ibuziuk** [comment]
@rhopp now it is `org.eclipse.che.api.core.ServerException: You are not allowed to start more workspaces.` :(

**Comment by ibuziuk** [comment]
[test]

**Comment by rhopp** [comment]
@ibuziuk That is not an actual failure. There is test trying to start more than one workspace (by intention). I've sent PR to get rid of this error message, which is causing this confusion: https://github.com/redhat-developer/rh-che/pull/1279

The failure on that run was again the project not being imported (even though the hard refresh of page should be already merged, right?) as seen on [this screenshot](http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-rh-che-prcheck-dev.rdu2c.fabric8.io/1218/screenshots/com.redhat.che.functional.tests.BuildAndRunProjectTest.runProjectBuild_time-1551774072985-millis.png) (btw. There is wrong link to artifacts here. Katka is wokring on fixing that)

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by rhopp** [comment]
I've rebased on master... hopefully it will help tests to pass ;-)

**Comment by ibuziuk** [comment]
@rhopp could you please investigate `Caused by: java.lang.Exception: Output: ; Error: must specify at least one container source.` test failures? We do need to stabilize PR check ASAP

**Comment by ibuziuk** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by amisevsk** [comment]
It looks like this PR is failing on *other* errors and does fix the postgres issue -- could we merge it without CI?

**Comment by ibuziuk** [comment]
@amisevsk agree, I just not aware of 3 different PRs for CI fix I thought this is the only one which should be build without issues. @rhopp WDYT ?

File: .ci/functional_tests_utils.sh [hunk 1]
```diff
@@ -98,5 +98,6 @@ function archiveArtifacts() {
   mkdir -p ./rhche/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
   cp ./logs/*.log ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
   cp -R ./logs/artifacts/screenshots/ ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
+	cp -R ./logs/artifacts/failsafe-reports/ ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
```
Comment (ibuziuk) [inline .ci/functional_tests_utils.sh]: @rhopp indent looks incorrect, + what is `/failsafe-reports` ?

Comment (rhopp) [reply .ci/functional_tests_utils.sh]: yep.. Indentation is wrong. I noticed that... I'll fix that before merging.
failsafe-reports is directory, where testng results are kept. But as I look at it, we still need some tweaking there: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-rh-che-prcheck-dev.rdu2c.fabric8.io/1254/failsafe-reports/

**Comment by rhopp** [comment]
[test]

**Pull request merged**